__tests__/__snapshots__/users.ts.snap

```diff
@@ -218,6 +219,7 @@ Object {
 exports[`query userReadingRank should return the reading rank 1`] = `
 Object {
   "currentRank": 2,
+  "lastReadTime": "2021-12-03T00:00:00.000Z",
```

Comment:
This is going to fail tomorrow probably. We need to use fixed dates. What do you think?

Reply:
I think should run smoothly since our expectation is `expect.anyhing`. I think this is due to the reason we compare the date today every time to see if it now matches the content of the query.

https://github.com/dailydotdev/daily-api/blob/0d0a998646ec08900a5623ca9f0363bbc48182b4/__tests__/users.ts#L270-L272

Reply:
And for the `lastReadTime`, we use the values in our `fixtures` for post which has static values.

Reply:
`lastReadTime` is not generated from the fixtures. It is calculated by the views we push to the database during the test. These views have relative timestamps because we want to test the ranking functionality. So I think you cannot test the `lastReadTime` as you do now. You have to compare it to the value not using snapshots but using the variables we define in the tests. Let me know if you want to talk about it

Reply:
Now, I understand. After checking the actual tests, this needs to have a separate test case. Thank you so much for elaborating very clearly 💎